src/help/h_developer.php

```diff
@@ -91,8 +90,7 @@ function print_settings() {
     }
 
     function print_submissions() {
-        echo "<p>The <code>api/paper</code> endpoint accesses conference
-submissions. GET calls return paper information; use
+        echo "<p>The <code>api/paper</code> endpoint accesses program submissions. GET calls return paper information; use
```

Comment:
Could you not change the line breaks but just the text? again easier for future merge.